### ISSUE TITLE
Screenshot is blue under Xwayland

### ISSUE THREAD

**Issue by wzin** [OttoAllmendinger/gnome-shell-screenshot#67]
Hello,

I've recently upgraded to 18.04 and started using Xwayland which is great.

Versions:

```
GNOME Shell 3.28.1
wayland 1.19.6-1ubuntu4
```

How to reproduce:

0. Log in to Ubuntu 18.04 using `Ubuntu on wayland`
1. install gnome screenshot tool via gnome tweaks or extensions.gnome.org
2. take screenshot
3. observe blue overlay on the screenshot as attached below:

![lpw3vta](https://user-images.githubusercontent.com/353944/39434158-15fa1868-4c98-11e8-8815-f37ff4b32e13.png)

The problem does not exist under Xorg.

**Comment by RicardoEPRodrigues** [comment]
I tested this and it didn't happen to me.

Pop_OS! 18.04 (based on Ubuntu 18.04) and Wayland.

**Comment by Iandenh** [comment]
I have the same problem. 
But the strange thing is that it is for the most programs it is only on the primary screen. Secondary screen is just fine.

But for Chrome the problem doesn't exist. There it is fine no matter which screen I take the screenshot.

**Comment by MikeStutzzzz** [comment]
Yeah tested on Pop as well. Primary Screen has blue covering it. If it is on Built in Screen with no external there is no issues

**Comment by pomke** [comment]
same problem, can't screenshot on 2nd screen, the screenshot is taken before the selection highlight clears.

**Comment by wzin** [comment]
![ylrnuun](https://user-images.githubusercontent.com/353944/40975916-c485a212-68cc-11e8-8b81-f654306567b5.png)

Replicated on Chrome - however it's not consistent. Can't tell if this is some kind of random race condition or if it depends on something.

I have  no problem screenshoting the 2nd screen.

**Comment by gitfineon** [comment]
Duplicate of https://github.com/OttoAllmendinger/gnome-shell-screenshot/issues/28

**Comment by OttoAllmendinger** [comment]
I submitted a new version containing the delay options to e.g.o.

Once it is approved, please report if the capture delay option mitigates the overlay problem.
